### PR TITLE
Assigment 2

### PR DESCRIPTION
**Pull request by krawcox**
For bonus I changed look of the tekst and I made it move.
For bonus for part 2 I just made objects overlap a screen.

### CODE REVIEW

**Comment by SamBumgardner** [comment]
The project crashes when built for the `neko` target with `lime test neko`. It does look like its working fine when built for HTML5, though.

I'm pretty sure it's happening because your PlayState is creating `Hero` objects upon initialization (when its first created with `new Playstate()` which happens before graphics-making systems are ready to run. 

Your hero objects try to access these systems when calling `makeGraphic()` so the game ends up crashing. 

We'll talk more about that issue during Monday's class, because it was happening in my example code as well.

**Comment by SamBumgardner** [comment]
You didn't add a `.gitignore` file to your project to exclude certain files from git, which means that your branch includes configuration for your IDE (the stuff in `.vscode` folder) and the executable results of building your project (everything inside the `export` folder). The assignment includes a link to an example `.gitignore` for this kind of project that will exclude those things.

This isn't a huge problem, but it means that your git repository now includes files (some of them very large, in the case of the `export` folder) that aren't useful or applicable to everyone that might clone your repository. 

Usually, you can expect that people cloning your repository are comfortable setting up their own IDE configuration (and not everyone's going to use `vscode`)

`export` should be excluded because the files in there can be large, they change after pretty much every build, and they're easily built after cloning the repository with something like `lime test neko`. There's not really a benefit to keeping them in your project, and it does make the project's history hard to read.

**Comment by SamBumgardner** [comment]
Nice work on the bonus challenges! 
Also, I like that I'm starting to learn a few polish words as I read your variable names 😄

File: source/Hero.hx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{
+    public var predkosc_x:Int;
+    public var predkosc_y:Int;
+    override public function new(x, y)
+        {
+            super();
+            var q:FlxRandom = new FlxRandom();
+            var p:FlxRandom = new FlxRandom();
+            this.predkosc_x = q.int(-5, 5);
+            this.predkosc_y = p.int(-5, 5);
```
Comment (SamBumgardner) [inline source/Hero.hx:19]: You typically only ever need one `FlxRandom` object for use in your game. After it's created, you can re-use it as many times as you need to generate random numbers.

We haven't talked about it in class yet, but one of the global-level utilities provided by `FlxG` is just a `new FlxRandom()` object, so to be more efficient you can just do `FlxG.random.int(-5, 5);` twice.
https://api.haxeflixel.com/flixel/FlxG.html#random

File: source/PlayState.hx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	{
+		tekst.x += 2;
+		tekst.y += 1;
+		bohater.update(elapsed);
+		bohater_1.update(elapsed);
+		bohater_2.update(elapsed);
+		bohater_3.update(elapsed);
+		bohater_4.update(elapsed);
+		bohater_5.update(elapsed);
+		bohater_6.update(elapsed);
```
Comment (SamBumgardner) [inline source/PlayState.hx:42]: All of your `Hero` objects have been added to the `PlayState` in lines `23 - 29`, so their update functions will automatically be called when the state does `super.update(elapsed)`. 

With these lines, your hero objects will actually be updating twice as often as they should. One update happens when you manually do it in these lines, and then they'll all get updated gain during the `super.update(elapsed)` call on line 43.

**Comment by krawcox** [comment]
I'm sorry I forgot to save changes to update and create. Now everything should be perfect

**Pull request merged**